clusterloader2/testing/density/scheduler-suite.yaml

```diff
@@ -1,4 +1,18 @@
+- identifier: vanilla
```

Comment:
Do we know how long the test will take at 5k node scale? What about the whole suite? If we don't please DM me, I'll help you with running the suite manually

I'm tentatively supportive for having the vanilla being part of the suite, but before I make the final call I'd like to have more data.

Reply:
According to past data: the throughput test (saturation deployment) is expected to take around ~25 minutes while the latency test is expected to take around ~11 hours. The test in total takes around ~13 hours, unfortunately :(

I think that's a bit much for a convenience feature, yes?

Reply:
Wow, 11 hours? Why does it take that long? The whole vanilla density test takes only 110 min  - https://k8s-testgrid.appspot.com/sig-scalability-kubemark#kubemark-5000&grid=beta&graph-metrics=test-duration-minutes
![FPAQY9mqKpFqgb4](https://user-images.githubusercontent.com/2604887/93179879-98148e00-f736-11ea-809f-c8c68c58c314.png)

Reply:
Ah nevermind, `testing/density/config.yaml` finishes in ~ 1h50m. The other 10+ hours is `testing/load/config.yaml`. Finally, `testing/access-tokens/config.yaml` also runs in the same job and it finishes in ~ 6m. I'm not fully aware of what `testing/load/config.yaml` does differently.

Since the scheduler suite uses `test/density/config.yaml`, I don't foresee any running time length issues with the 5k job :+1: